src/com/sfuentes/MedianAndSmallest.java

```diff
@@ -0,0 +1,24 @@
+package com.sfuentes;
+
+public class MedianAndSmallest {
```

Comment:
Also, here you need to change the name of the class because it still says Median